src/test/groovy/com/synacy/gradprogram/spock/exercise/DeliveryServiceSpec.groovy

```diff
@@ -0,0 +1,50 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+
+class DeliveryServiceSpec extends Specification {
+
+    DeliveryService service
+
+    DateUtils dateUtils = Mock(DateUtils)
+    DeliveryRequestRepository deliveryRequestRepository = Mock(DeliveryRequestRepository)
+
+    def setup() {
+        service = new DeliveryService(dateUtils, deliveryRequestRepository)
+    }
+
+    def "createDelivery should be able to save the created delivery request."(){
+        given:
+        Order order = new Order()
+
+        UUID id = order.getId()
+        DateUtils deliveryDate = dateUtils.getCurrentDate()
```

Comment:
You mean to use `Date` here not `DateUtils`? Also the value for this will just be `null` since the `dateUtils` is a Mock and we didn't stub the `getCurrentDate()` method.